src/cljs/athens/views/blocks/core.cljs

```diff
@@ -361,11 +483,17 @@
            [presence/inline-presence-el uid]
 
            (when (and (> (count _refs) 0) (not= :block-embed? opts))
-             [block-refs-count-el (count _refs) uid])]
+             [block-refs-count-el (count _refs) #(swap! state update :inline-refs/open not)])]
```

Comment:
If normal click, open inline refs. **If shift-click, open in right sidebar.**

Reply:
fixed